pegasus/sites.v3/code.org/public/learn2016/index.haml

```diff
@@ -14,6 +15,36 @@ social:
 
 = view 'mobilecheck.js'
 
+#fullwidth{style: "background-color: #2799a4; overflow: hidden"}
+  .thegap{style: "height: 1px"}
+    &nbsp;
+  #desktop-header.desktop-feature
+    = view :header, variant: "desktop"
+  #mobile-header.mobile-feature
+    = view :header, variant: "mobile"
+
+  .banner
+    .container_responsive{style: "color: white"}
+      .col-50{style: "text-align: center"}
+        %img{src: "/images/learn/hour-of-code-logo-2016.png", style: "margin-top: 30px; margin-bottom: 30px"}
```

Comment:
Not blocking since it's mostly for i18n but later let's use actual text instead of an image.